tianshou/env/pettingzoo_env.py

```diff
@@ -0,0 +1,112 @@
+from abc import ABC
+from typing import Any, Dict, List, Tuple
+
+import gym.spaces
+from pettingzoo.utils.env import AECEnv
+from pettingzoo.utils.wrappers import BaseWrapper
+
+
+class PettingZooEnv(AECEnv, ABC):
```

Comment:
```suggestion
class PettingZooEnv(AECEnv, gym.Env, ABC):
```
so that we don't need to have a special check in collector?

Reply:
Agree.